starwars/src/components/StarWarsList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import StarWarsData from './StarWarsData';
+import './StarWars.css';
+
+function StarWarsList (props) {
+    return(
+        <div>
+            {props.StarWarsData.map(chars => 
+                <StarWarsData key={chars.name} chars={chars} />
+            )}
+        </div>
```

Comment:
on line 8 you are mapping over the state you passed from the App.js. It looks like you named state 'Chars'... so that is what you should be mapping over. corrected of course to lowercase.